access_group_cache/main.py

```diff
@@ -121,10 +121,24 @@ def index():
         cpg_utils.cloud.read_secret(ANALYSIS_RUNNER_PROJECT_ID, 'server-config')
     )
 
+    group_types = [
+        'access',
+        'web-access',
+        'test',
+        'standard',
+        'full',
+    ]
+    # add SM group types
+    group_types.extend(
+        f'sample-metadata-{env}-{rs}'
+        for env in ('production', 'test')
```

Comment:
Should we name this `main` and `test`, for symmetry with the buckets?

The "opposite" of _prod_ would be _dev_, which is not what you mean here, IIUC. I.e. test projects like `seqr-test` would still be in the production instance.

Reply:
Yes, totally agree! Finished this rename across both files.